.cascade-code/Chapman.edu/test-section/tatwell/fast-omni-nav/omni_nav.vtl

```diff
@@ -0,0 +1,514 @@
+#*
+ * OmniNav format refactored to work with faster loading index block.
+ * Chapman.edu/test-section/tatwell/fast-omni-nav/omni_nav_format.vtl
+ *
+ * Refactor reduced format from about 630 lines to 515.
+ *
+ * TODO: Move off-canvas-nav macros into their own file.
+ *#
+
+#import( "/_cascade/formats/helpers/debug.velocity" )
+
+##
+## Hardcoded Params
+##
+## umbrellaCategories
+## These are locations for which OmniNav should build the secondary off-canvas-nav menu.
+## umbrellaCategory is the term product owners use.
+## Does order matter here? I'm assuming it does.
+## Inner Array: [ directory (assume leading /), menu label, class/image id, path ]
+#set ( $umbrellaCategories = [
+  [ 'business', 'ARGYROS SCHOOL', 'asbe', 'business/index.aspx' ],
+  [ 'education', 'ATTALLAH COLLEGE', 'education', 'education/index.aspx' ],
+  [ 'dodge', 'DODGE COLLEGE', 'dodge', 'dodge/index.aspx' ],
+  [ 'crean', 'CREAN COLLEGE', 'crean', 'crean/index.aspx' ],
+  [ 'wilkinson', 'WILKINSON COLLEGE', 'wilkinson', 'wilkinson/index.aspx' ],
+  [ 'copa', 'CoPA', 'copa', 'copa/index.aspx' ],
+  [ 'pharmacy', 'CUSP', 'cusp', 'pharmacy/index.aspx' ],
+  [ 'law', 'FOWLER SCHOOL OF LAW', 'fowler', 'law/index.aspx' ],
+  [ 'scst', 'SCHMID COLLEGE', 'schmid', 'scst/index.aspx' ],
+  [ 'about', 'About', 'default', '/' ],
+  [ 'academics', 'Academics', 'default', '/' ],
+  [ 'admission', 'Admission', 'default', '/' ],
+  [ 'alumni', 'Alumni', 'default', '/' ],
+  [ 'campus-services', 'Campus Services', 'default', '/' ],
+  [ 'faculty-staff', 'Faculty and Staff', 'default', '/' ],
+  [ 'families', 'Families', 'default', '/' ],
+  [ 'research', 'Research', 'default', '/' ],
+  [ 'students', 'Students', 'default', '/' ],
+  [ 'support-chapman', 'Support Chapman', 'default', '/' ],
+  [ 'communication', 'SCHOOL OF COMM', 'soc', 'communication/index.aspx' ]
+])
```

Comment:
4 lists from original one consolidate in a list of lists. Considered a map, but I wasn't sure order mattered. Anyway, this is probably simpler.